.circleci/config.yml

```diff
@@ -15,6 +15,9 @@ executors:
           FXA_EMAIL_ENV: dev
           FXA_EMAIL_LOG_LEVEL: debug
           RUST_BACKTRACE: 1
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
```

Comment:
Where are these define? I guess they'll be here? https://app.circleci.com/settings/project/github/mozilla/fxa/environment-variables?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fmozilla%2Ffxa

Reply:
Nah, they're actually def'd in the circleci env

Reply:
isn't that what i said?